pkg/runner/runner.go

```diff
@@ -287,6 +301,26 @@ func (r *Runner) deployAssets() error {
 			return err
 		}
 	}
+	if r.gatewayApi {
+		log.Debugf("Creating GatewayClass...")
+		_, err = hrClientSet.GatewayV1beta1().GatewayClasses().Create(context.TODO(), gatewayClass, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+		time.Sleep(5 * time.Second) // wait for ServiceMeshControlPlane to be ready
+		log.Debugf("Creating Gateway...")
+		_, err = hrClientSet.GatewayV1beta1().Gateways(string(gatewayNamespace)).Create(context.TODO(), &gateway, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+		log.Debugf("Waiting 4 minutes for Gateway to be ready...")
+		time.Sleep(4 * time.Minute)
```

Comment:
Why so long?

Reply:
it takes ~3mins for the gateway to be up